eZ/Publish/API/Repository/Tests/FieldType/CheckboxIntegrationTest.php

```diff
@@ -392,6 +387,21 @@ public function testFindContentFieldCriterion(bool $isActive): void
         self::assertSame($isActive, $value->bool);
     }
 
+    public function testAddFieldDefinition()
+    {
+        $content = $this->addFieldDefinition();
+
+        $this->assertCount(2, $content->getFields());
+
+        $this->assertFalse(
+            $this->getRepository()->getFieldTypeService()->getFieldType(
```

Comment:
Readability, please extract
```php
$fieldTypeService = $this->getRepository()->getFieldTypeService()
```
as the first call of the method. If you need to call it after `addFieldDefinition` then it should be called as `getRepository(false)` because the method has side effects.